.github/workflows/pr.yaml

```diff
@@ -199,7 +199,9 @@ jobs:
           if (length(repo) == 0) {
              break
           }
-          lnt <- lint_dir(repo, relative_path=T, linters=linters)
+          lnt <- lint_dir(repo, relative_path=T, linters=linters, exclusions=list("tools/SRS_tools/comparison_div.r"=153, "tools/SRS_tools/comparison_div.r"=160, "tools/SRS_tools/Lib_preprocess_S2.r"=76, "tools/SRS_tools/Lib_preprocess_S2.r"=77, 
+          "tools/SRS_tools/Lib_preprocess_S2.r"=79, "tools/SRS_tools/Lib_preprocess_S2.r"=80, "tools/SRS_tools/Lib_preprocess_S2.r"=81, "tools/SRS_tools/Lib_preprocess_S2.r"=84, "tools/SRS_tools/Lib_preprocess_S2.r"=85,
```

Comment:
why do you exclude this here instead of fixing the R code? Is that not possible?

Reply:
I corrected some of the linting issues that could be fixed (that I forgot to update). However, for "comparison_div.r" I cannot fix it or the code does not work as well and for the 2 other files it is not my code I fixed most of the linting issues. For these last files I let some issues because I think it makes more sense as it is. I can fix them if you think I should.